setup.py

```diff
@@ -28,6 +28,6 @@
     setup_requires=[
         'pbr>=1.8',
         'requests!=2.8.0,>=2.5.2',
-        'requests-unixsocket==0.1.4',
+        'requests-unixsocket>=0.1.5',
```

Comment:
Could you add a comment as to why we need `0.1.5`? That would make it clear that there's a hard dependency here (whereas, `0.1.4` was there because it was the released version when we first started using `request-unixsocket`. Maybe something like `# Needed for HTTP_PROXY support`